.travis.yml

```diff
@@ -15,61 +15,16 @@ env:
   - TEST=tvOS
   - TEST=iOS-Example
   - TEST=SPM
+  - BUILD="TARGET=RxSwift SWIFT_VERSION=5.0 ./scripts/validate-podspec.sh"
+  - BUILD="TARGET=RxCocoa SWIFT_VERSION=5.0 ./scripts/validate-podspec.sh"
+  - BUILD="TARGET=RxBlocking SWIFT_VERSION=5.0 ./scripts/validate-podspec.sh"
+  - BUILD="TARGET=RxTest SWIFT_VERSION=5.0 ./scripts/validate-podspec.sh"
+
 matrix:
     exclude:
       - os: osx
         env: DANGER=1
 
-    include:
-      - osx_image:
-        os: linux
-        env: TEST=Unix swift=5.0
```

Comment:
I don't see any Linux target. We should have at least:
```
    include:	
      - osx_image:	
        os: linux	
        env: TEST=Unix swift=5.0
```

as far as I can tell.